app/helpers/user_discussions_helper.rb

```diff
@@ -0,0 +1,38 @@
+module UserDiscussionsHelper
+  def user_discussions_table_title(discussion, user, last_read)
+    %Q{
+      <tr></tr>
+      <thead>
+        <tr>
+          <td class="#{last_read.nil? ? '' : 'pt-5'}" colspan="4">
+            <strong>#{discussion.read_by?(@user) ? t(:discussions_read) : t(:discussions_unread)}</strong>
```

Comment:
You ate both using a param and an attribute. Is that ok?

Reply:
Ah, good catch. No, it's not ok. I mean, it works, because the attributes exist in the view, so the helper knows about them, but then it makes no sense to pass the params as well.

I'm changing it to params only so it's consistent with `discussion` (that one can't be an attribute).